x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/scalar/geo/GeoShape.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+package org.elasticsearch.xpack.sql.expression.function.scalar.geo;
+
+import org.elasticsearch.common.geo.builders.ShapeBuilder;
+import org.elasticsearch.common.geo.parsers.ShapeParser;
+import org.elasticsearch.common.xcontent.ToXContentFragment;
+import org.elasticsearch.common.xcontent.XContentBuilder;
+
+import java.io.IOException;
+
+/**
+ * Wrapper class to represent a GeoShape in SQL
+ */
+public class GeoShape implements ToXContentFragment {
```

Comment:
Out of curiosity, is this class necessary for the SQL plugin? It seems to mostly be a wrapper?

Reply:
The wrapper overrides the XContent serialization to use WKT instead of GeoJSON, which is default representation on the SQL side. So, now that I am thinking about it I am not sure if it makes sense to move this to sql package as we previously discussed.

Reply:
Okay, no problem. We can take a closer look at moving things around after we get BKD backed shapes committed so I'm not too worried about this atm.